packages/server/migration/1616985857679-HideInsights.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class HideInsights1616985857679 implements MigrationInterface {
+  name = 'HideInsights1616985857679';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
can we merge this and `AddInsights`? since it's all in one PR now

Reply:
Yep, just merged it.